templates/store/saga/index.ts

```diff
@@ -0,0 +1,81 @@
+import { all, takeLatest, put, call, fork } from 'redux-saga/effects';
+{{setVar "capitalizedName" (capitalize name)}}
+import { getArticlesService } from '@core/services';
+import {
+  {{capitalizedName}}Types,
+  fetch{{capitalizedName}}Success,
+  reload{{capitalizedName}},
+  fetch{{capitalizedName}}Failed,
+} from '../actions';
+{{setVar "upperCaseName" (toUpperCase name)}}
+
+export type dateValues = '' | 'today' | 'week' | 'month';
```

Comment:
I do not think if we should keep all CRA-related references in this template- let's just have dummy generators that do nothing in fact, instead of relying on CRA